packages/jest-emotion/README.md

```diff
@@ -57,6 +58,21 @@ expect.addSnapshotSerializer(
   })
 )
 ```
+
+### `disableDOMElements`
```

Comment:
Could you rename this option to `DOMElements` and make it so that it's true by default and making it false disables printing DOM elements' styles.

Reply:
done